src/test/resources/dataPopulation/mittente_1.yaml

```diff
@@ -0,0 +1,10 @@
+url: "https://selfcare.dev.notifichedigitali.it"
+user: "ggiorgi"
+pwd: "test"
```

Comment:
not use yaml file, pass all data in tables cucumber

Reply:
done

Reply:
done